docs/91_version_history.rst

```diff
@@ -33,175 +33,111 @@ Release Date: 2025-03-13
 
 
     - Bug Fix:
 
-       - 
-       - 
-       - The error handling for the rules configuration of the asset filter has been improved.
-       - 
-       - fledge-south-s2opcua: for the _Asset Naming Scheme_ selection of _Single Datapoint_ or _Single Asset_, Browse Names of Variables must be unique among all OPC UA Subscriptions. If there are duplicate Browse Names, the plugin should disambiguate the names by concatenating the Variable’s Node Id to the end of the Browse Name. If there are more than 2 duplicate Browse Names, the plugin would concatenate the Node Id too many times. This has been fixed.
-       - 
-       - 
-       - 
-       - 
-       - Add support for the Suez Water south plugin on Ubuntu 22.
-       - 
-       - 
-       - 
-       - 
-       - 
-       - 
-       - 
-       - 
-       - 
-       - 
-       - A problem that could result in excessive memory use when the delta filter is used.
-       - 
-       - 
-       - 
-       - 
-       - An issue with the dynamic reconfiguration of the observer plugin which altered the measurement filtering criteria has been resolved. Previously the new filtering rules did not immediately have an impact and a restart ofthe service was the only reliable means to ensure the new rules were fully implemented.
-       - Release Notes are in [FOGL-8895|https://scaledb.atlassian.net/browse/FOGL-8895]
-       - Release Notes are in [FOGL-8895|https://scaledb.atlassian.net/browse/FOGL-8895]
-       - The plugin developers guide has been updated to include more information on the 2.0.0 version of the south plugin interface.
-       - 
-       - Do not include in the release notes. The fix for this problem is documented in the Sanitized Description of FOGL-9181.
-       - OMF North plugin: sending OMF Data messages to the Edge Data Store 2020 resulted in the HTTP error code 400 with the message \_One or more errors occurred. (The action 'Update' is not supported for OMF messages.)_.\ The same problem occurs when sending OMF Data messages to the AVEVA Connector Relay. This has been fixed. This problem does not occur in EDS 2023, EDS 2023 Patch 1 and EDS 2024.
-       - 
-       - 
-       - 
-       - An issue that would occasionally cause the addition of the Observer south plugin to a disabled service to crash has been resolved.
+       - fledge-south-s2opcua: for the Asset Naming Scheme selection of Single Datapoint or Single Asset, Browse Names of Variables must be unique among all OPC UA Subscriptions. If there are duplicate Browse Names, the plugin should disambiguate the names by concatenating the Variable’s Node Id to the end of the Browse Name. If there are more than 2 duplicate Browse Names, the plugin would concatenate the Node Id too many times. This has been fixed.
+       - An issue with the fledge-south-mqtt-sparkplug plugin that would allow illegal asset names to be altered has been resolved.
```

Comment:
This must be removed, as it is mandatory since version 2.6.0, and the issue was identified only in the 2.6.0_next version.